src-java/floodlightrouter-topology/floodlightrouter-storm-topology/src/main/java/org/openkilda/wfm/topology/floodlightrouter/bolts/SpeakerToControllerProxyBolt.java

```diff
@@ -93,14 +93,16 @@ protected void proxyInfoMessage(String key, InfoMessage envelope) {
         InfoData payload = envelope.getData();
         if (payload instanceof IslInfoData) {
             proxyOnlyIfActiveRegion(key, envelope, (IslInfoData) payload);
-        } else if (payload instanceof PortInfoData) {
-            proxyOnlyIfActiveRegion(key, envelope, (PortInfoData) payload);
         } else if (payload instanceof IslOneWayLatency) {
             proxyOnlyIfActiveRegion(key, envelope, (IslOneWayLatency) payload);
         } else if (payload instanceof IslBaseLatency) {
             proxyOnlyIfActiveRegion(key, envelope, (IslBaseLatency) payload);
         } else if (payload instanceof ConnectedDevicePacketBase) {
             proxyOnlyIfActiveRegion(key, envelope, (ConnectedDevicePacketBase) payload);
+        } else if (payload instanceof PortInfoData) {
+            log.error(
```

Comment:
is it really error level?

Reply:
Yes, it should be error. If this event will arrave into this bolt/handler, something have goes wrong in event routing - PortInfoData must be handled by `SpeakerToNetworkProxyBolt` only.